packages/generator-particle-patternlab/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "outDir": "lib",
+    "rootDir": "./src",
+    "strict": true,
```

Comment:
We may want to bring `strict` to the base of the project. This may require additional fixes. 

I did run this in `eleven` and found no errors. So we're probably good to go with moving this to the base tsconfig.json in the root of the project.